pyro/ops/integrator.py

```diff
@@ -56,30 +46,38 @@ def single_step_velocity_verlet(z, r, potential_fn, inverse_mass_matrix, step_si
     """
     z_next = z.copy()
     r_next = r.copy()
-    z_grads = _potential_grad(potential_fn, z_next)[0] if z_grads is None else z_grads
+    return _single_step_inplace_verlet(z_next, r_next, potential_fn, inverse_mass_matrix, step_size, z_grads)
```

Comment:
nit: we might call `velocity_verlet` with `num_steps=1` here

Reply:
Good point. Yeah, from what I can see, we now don't need `single_step_velocity_verlet`; will refactor.